Source/WebCore/animation/KeyframeEffect.h

```diff
@@ -26,6 +26,7 @@
 #pragma once
 
 #include "AcceleratedEffect.h"
+#include "Animation.h"
```

Comment:
Why is this change necessary?

Reply:
Moved build fixes to https://github.com/WebKit/WebKit/pull/39640.